src/services/cooperation.js

```diff
@@ -69,17 +71,8 @@ const cooperationService = {
       throw createError(409, VALIDATION_ERROR('You can change only either the status or the price in one operation'))
     }
 
-    const cooperation = await Cooperation.findById(id).exec()
-    if (!cooperation) {
```

Comment:
Removed this check because we already have middleware in place for this endpoint that performs the same validation, but returns a 400 error instead:
```
{
  "status": 400,
  "code": "INVALID_ID",
  "message": "ID is invalid."
}
```